AirCasting/Utils/DataFormatters.swift

```diff
@@ -2,9 +2,47 @@
 //
 
 import Foundation
+import Resolver
 
 enum DateFormatters {
     
+    struct SessionCardView {
+        @InjectedObject private var userSettings: UserSettings
+        static let shared = SessionCardView()
```

Comment:
this is ok, but for singletons we additionally want to disable the possibility to create `SessionCardView`. Right now there is no problem with doing that in the rest of the codebase (just `SessionCardView()`). To do so we make the `init` private:

```Swift
private init() { }
```